packages/ra-core/src/dataProvider/useUpdate.ts

```diff
@@ -382,7 +382,7 @@ export const useUpdate = <RecordType extends RaRecord = any>(
         }
     };
 
-    return [update, mutation];
```

Comment:
You don't need to pass refs in dependencies

Reply:
@djhi 
Sorry, I marked draft and I intended to revise but, I couldn't figure out how useUpdate cache previousData.
Now, I understand lifecycle in useQuery makes record changed and form's record nothing relevant, so at least one changes by getOne, so we can use record variable.
But, I thought this is very confused so, I make the variable with clearer name as cachedData. I removed draft sign. So could you review again?